src/app.html

```diff
@@ -4,9 +4,10 @@
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
 		<meta name="viewport" content="width=device-width" />
+		<link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap" rel="stylesheet" />
 		%sveltekit.head%
 	</head>
-	<body data-sveltekit-preload-data="hover" class="bg-[rgb(18,22,22)] bg-[url('/bg.jpg')] h-screen w-screen bg-center bg-cover text-white">
+	<body data-sveltekit-preload-data="hover" class="bg-[#15171B] h-screen w-screen bg-center bg-cover text-white">
```

Comment:
this can be added as a custom color in the tailwind config instead of passing the hex (only worth it if we want to reuse it down the line)